book/06-github/sections/3-maintaining.asc

```diff
@@ -1,134 +1,133 @@
+Hindi namin balabaan ito dito; kung kailangan mo ng refresher, i-check out ang <<_git_basics_chapter#_git_basics_chapter>>.
 
-Now that your project is hosted on GitHub, you can give the URL to anyone you want to share your project with.
-Every project on GitHub is accessible over HTTPS as `https://github.com/<user>/<project_name>`, and over SSH as `git@github.com:<user>/<project_name>`.
-Git can fetch from and push to both of these URLs, but they are access-controlled based on the credentials of the user connecting to them.
+Ngayon na naka-host na ang iyong proyekto sa GitHub, maaari kang magbigay ng URL sa sinuman na gusto mong bahagian ng iyong proyekto.
+Bawat proyekto sa GitHub ay maaaring i-access sa HTTPS bilang `https://github.com/<user>/<project_name>`, at sa SSH bilang `git@github.com:<user>/<project_name>`.
+Maaaring mag-fetch ang Git mula sa at mag-push sa parehong mga URL na ito, ngunit sila ay kontrolado ng access batay sa mga kredensyal ng gumagamit na kumukonekto dito.
 
-[NOTE]
+[TANDAAN]
 ====
-It is often preferable to share the HTTPS based URL for a public project, since the user does not have to have a GitHub account to access it for cloning.
-Users will have to have an account and an uploaded SSH key to access your project if you give them the SSH URL.
-The HTTPS one is also exactly the same URL they would paste into a browser to view the project there.
+Madalas na lalong kanais-nais na ibahagi ang URL na nakabatay sa HTTPS para sa isang pampublikong proyekto, dahil ang gumagamit ay hindi kailangang magkaroon ng isang account sa GitHub upang ma-access ito para sa pag-clone.
+Ang mga gumagamit ay magkakaroon ng isang account at isang na-upload na SSH key upang ma-access ang iyong proyekto kung binibigyan mo sila ng SSH URL.
+Ang HTTPS ay isa ring eksaktong parehong URL na kanilang idikit sa isang browser upang tingnan ang proyekto doon.
 ====
 
-==== Adding Collaborators
+==== Pagdaragdag ng mga Tagapangasiwa
 
-If you're working with other people who you want to give commit access to, you need to add them as ``collaborators''.
-If Ben, Jeff, and Louise all sign up for accounts on GitHub, and you want to give them push access to your repository, you can add them to your project.
-Doing so will give them ``push'' access, which means they have both read and write access to the project and Git repository.
+Kung ikaw ay nagtatrabaho kasama ang ibang tao nga nais mong bigyan ng access sa pag-commit, kailangan mong idagdag sila bilang ``tagapangasiwa''.
+Kung si Ben, Jeff, and Louise ay nag-sign up ng mga account sa GitHub, at gusto mo silang bigyan ng access sa pag-push sa iyong repositoryo, maaari mo silang idagdag sa iyong proyekto.
+Paggawa nito ay nagbibigay sa kanila ng access sa ``push'', na nangangahulugan na mayroon silang access sa pagbasa at pagsulat sa proyekto at  repositoryo ng Git.
```

Comment:
- you could change `Paggawa` to `Ang paggawa`
- spacing between `at` and  `repositoryo`